### PR TITLE
further rewriting

### PR DESCRIPTION
**Pull request by ungoldman**
When y'all have a moment, I did some further work on the style. Opinions welcome.

@paulcpederson @nikolaswise

### CODE REVIEW

**Comment by ungoldman** [comment]
protip: `build` is a reserved npm command/script -- https://docs.npmjs.com/cli/build

So if you want to make a generic building task it's often a good idea not to name it build. I learned the hard way and broke the deployment on a project once.

File: src/_fonts.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+               'lucida grande', /* Older MacOS */
+               'helvetica neue', helvetica, /* Older MacOS */
+               'Fira Sans', /* Firefox OS */
+               roboto, noto, /* Google stuff */
+               'Droid Sans', /* Old Google stuff */
+               cantarell, oxygen, ubuntu, /* Linux stuff */
+               'franklin gothic medium', 'century gothic', /* Windows stuff */
+               sans-serif; /* Everything else */
+
+$system-code-fonts: Consolas,
```
Comment (paulcpederson) [inline src/_fonts.scss]: https://github.com/ungoldman/style.css/issues/9#issuecomment-246092967 is an idea, but I think what you have is pretty solid. For me, Menlo should be first but I'm biased by my editor probably.

File: package.json [hunk 2]
```diff
@@ -27,18 +28,21 @@
...
   },
   "scripts": {
-    "minify": "node minify.js",
-    "sass": "node-sass src/ -o ./",
-    "watch:sass": "node-sass src/ -o site/ -w",
-    "build:site": "rm -rf site/* && sitedown . -b site/ -l layout.html --pretty=false && cp style* site",
-    "serve": "static site -p 8000",
-    "prestart": "npm run sass && npm run build:site",
-    "start": "npm-run-all --parallel serve watch:sass",
+    "prestart": "rm -rf site/* && npm-run-all --parallel site:*",
```
Comment (paulcpederson) [inline package.json]: dope

**Comment by ungoldman** [comment]
Merging for now. Still on the fence about serif/sans. Also plenty of work left to do for details like tables, figures, definition lists, etc.

**Pull request merged**